test/v2/controllers/V2MovementsControllerSpec.scala

```diff
@@ -978,6 +1020,17 @@ class V2MovementsControllerSpec
             eqTo(Some(MovementType.Departure)),
             eqTo(Some(MessageType.DeclarationData))
           )(any[HeaderCarrier], any[ExecutionContext])
+
+          verify(mockAuditService, times(1)).auditStatusEvent(
+            eqTo(SubmitDeclarationFailed),
+            eqTo(Some(Json.obj("message" -> "Internal server error", "code" -> "INTERNAL_SERVER_ERROR"))),
```

Comment:
Same here no need to construct json object, just use ```Json.toJson(PresentationError.internalServiceError())```

Reply:
same